crates/sql_provider_datafusion/src/dbconnectionpool.rs

```diff
@@ -0,0 +1,31 @@
+use std::{collections::HashMap, sync::Arc};
+
+use spicepod::component::dataset::acceleration;
```

Comment:
This is a weird dependency for a generic sql_provider_datafusion crate to have.

Reply:
Good point, moved into spicepod